openshift/redis-persistent.json

```diff
@@ -65,6 +65,25 @@
       }
     },
     {
+      "kind": "PersistentVolume",
+      "metadata": {
+        "name": "pv${PV}"
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteOnce",
+          "ReadWriteMany"
+        ],
+        "capacity": {
+          "storage": "10Gi"
+        },
+        "persistentVolumeReclaimPolicy": "Recycle",
+        "hostPath": {
```

Comment:
This makes the template not really reusable between installations. PersistentVolumes are usually not created in application templates. Using `hostPath` will lock the deployment to one node and is not production ready.